### ISSUE TITLE
Some questions and a suggestion

### ISSUE THREAD

**Issue by vinsis** [cypw/PyTorch-MFNet#1]
Hi Yunpeng, just read your paper and have a couple of quick questions:

* Why do you implement batchnorm + relu before the convolution in the `BN_AC_CONV3D` class [here](https://github.com/cypw/PyTorch-MFNet/blob/master/network/mfnet_3d.py#L27-L28)?

* In your paper you say: `we
set  the  number  of  the  first-layer  output  channels  to  be  4  times  smaller  than
its  input  channels, ... `
Hence, shouldn't this [line](https://github.com/cypw/PyTorch-MFNet/blob/master/network/mfnet_3d.py#L36) be written as shown below?

```
# current version: num_ix = int(num_mid/4)
num_ix = int(num_in/4)
```
I believe `self.conv_i1` and `self.conv_i2` are the layers for the multiplexer. Or am I getting it wrong?

* This is a theoretical question: is the role of multiplexer to rearrange information across channels so as to minimize information loss for each fiber? 

Lastly, a small suggestion: since you are using PyTorch v0.4, you need not use `Variable` anymore. Hence you can write this [line](https://github.com/cypw/PyTorch-MFNet/blob/master/network/mfnet_3d.py#L185) as:

`data = torch.randn(1,3,16,224,224)`

Thank you.

**Comment by cypw** [comment]
Hi @vinsis,

Thank you for the questions.

* We put "batchnorm + relu" before each convolution layer since it is more clear and easier to design. It is actually following [this](https://arxiv.org/abs/1603.05027) paper from Kaiming. We haven't tried putting them after the convolution, but it should give the same or even better performamce, see [here](https://github.com/tornadomeet/ResNet#resnet-v2-vs-resnet-v1).

* You are right. Please refer to the code for the correct version. We will update the paper soon. Thank you for pointing it out.

* Multiplexer can pass information across fibers, so that features learned by one fiber can be seen by other fibers. It has the effect of increasing the overall learning capacity and minimize information loss.

* Thank you for the suggestion.

Thank you.

**Comment by vinsis** [comment]
Thanks @cypw for replying. Closing the issue now.
